### ISSUE TITLE
VAMC Health Services template: adopt V1 & v3  components (incl spotlights) where components aren't used

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#17657]
## Description
#16997 tracked adopting existing V3 components

VAMC Health Service pages need the following updates to components:

Example pages:

- [VAMC Boston Health Services](https://www.va.gov/boston-health-care/health-services/)
- [VAMC Minneapolis health Services](https://www.va.gov/minneapolis-health-care/health-services/)

### Elements that are not using components that should be

1. Link
   - Links are not using a component but should be using [the v1 component](https://design.va.gov/components/link/) if possible, with the understanding that CMS content has limitations
2. ~Telephone~
   - ~[The component](https://design.va.gov/components/telephone) should be used if possible, with the understanding that CMS content has limitations~
   - ~Exts in phone numbers should be parsed correctly~
3. Back to top
   - The page meets the criteria guidelines for the [Back to top component](https://design.va.gov/components/back-to-top) I believe so this should be added
4. Spotlights
   - We are updating the spotlights to use [the card component](https://design.va.gov/components/card#default---white-with-border). 
   - This will include: 
     - Outline
     - bg color
     - header level
   - Will NOT include the link treatment in card screenshot below.
   - Future state: ~[Figma design with full annotation notes.](https://www.figma.com/file/ikCp2eOubIgbN1TwyEYIrN/VAMC-Master?type=design&node-id=428%3A3013&mode=design&t=aRQ2J73hbbiuFEsR-1)~
   
![Screenshot 2024-03-22 at 12 04 32 PM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/137226108/6f08d218-9110-433a-ae95-a6913fc51ab7)

## Acceptance Criteria
- [x] Link component has been added
- [x] Back to top component has been added, if applicable
- [x] Card component, with correct markup has been added to Spotlights
- [x] Update and run existing test(s).
- [x] If additional tests are needed, stub a ticket to note this for future work.
- [x] Requires accessibility review
- [ ] When PR is ready to merge, @jilladams for change management trigger
- [ ] Change management for Spotlight update
- [x] Backport findings re: Spotlight grid to #18126

**Comment by laflannery** [comment]
I added an AC for change management for the spotlight section - I spoke to @mmiddaugh and since there is a large-ish display chang she felt that giving folks a heads up that this was happening would be good.

**Comment by laflannery** [comment]
Telephone moved to #16148

**Comment by jilladams** [comment]
We don't want naked links. 
To achieve that: create the Spotlight section in a way that they'll all have an equal width when there are multiple spotlights.  (#18126  offers a potential solution.)

**Comment by laflannery** [comment]
@jilladams - updated your comment above with just some more specific context

**Comment by randimays** [comment]
I saw the change management discussion in Slack here: https://dsva.slack.com/archives/C06NMUK12LR/p1716323803156549. Please let me know if there are any other things we need to do from an engineering perspective.

The code has been merged to production and validated successfully. Closing!
